run.sh

```diff
@@ -15,9 +45,9 @@ if [[ $? -eq 0 ]]; then
 
     str_num=$(echo $status | sed -nr "s/.*на странице №([[:digit:]]{1,4})!.*/\1/p")
     echo "Задача $zad найдена на странице №$str_num!"
+    
+    open $sem $str_num
 
 else
     echo "Задача $zad не найдена в корявнике:("
```

Comment:
Consider the following possibility: if Koryavov doesn't have the necessary task you can make a request to open one of the other sources (e.g. the corresponding Google-drive with solved tasks or Botay source). I am ready to provide all the necessary links if it necessary.